node/src/accountant/test_utils.rs

```diff
@@ -889,90 +889,131 @@ impl PendingPayableDaoFactoryMock {
     }
 }
 
-pub type PayableScannerBuilder = PayableScanner;
-
-impl Default for PayableScannerBuilder {
-    fn default() -> PayableScanner {
-        PayableScanner::new(
-            Box::new(PayableDaoMock::new()),
-            Box::new(PendingPayableDaoMock::new()),
-            Rc::new(PaymentThresholds::default()),
-        )
-    }
+pub struct PayableScannerBuilder {
```

Comment:
Since we are calling the builder as `AccountantBuilder` and using it only for tests. I think it's appropriate to use this name.

I personally prefer short names, whenever possible :)